tests/app/main/views/organizations/test_organization_invites.py

```diff
@@ -262,15 +271,15 @@ def test_registration_from_org_invite_404s_if_user_not_in_session(
             {
                 "name": "Bad Mobile",
                 "mobile_number": "not good",
-                "password": "validPassword!",
+                "password": "validPassword!",  # noqa
```

Comment:
`# noqa` needed because of a reference to a key named `password`?

(Same question for the other occurrences of this.)

Reply:
It's the new .ds.baseline that searches out test code for passwords. :(

Reply:
Yup, figured as much, just wanted to be sure!